runauto_nag.sh

```diff
@@ -2,21 +2,42 @@
 PATH_SCRIPT="$( cd "$(dirname "$0")" ; pwd -P )"
 . $PATH_SCRIPT/venv/bin/activate
 cd $PATH_SCRIPT
+# Nag to developers
+# Daily
```

Comment:
what does the "daily" mean here?  How is it different than the other checks?

Reply:
Others are conditional, this one is always sent